app/jinja_filters.py

```diff
@@ -21,9 +21,9 @@ def format_percentage(value):
 
 @evalcontextfilter
 @blueprint.app_template_filter()
-def nl2br(context, value):
+def format_multilined_string(context, value):
     _paragraph_re = re.compile(r'(?:\r\n|\r|\n){2,}')
-    result = u'\n\n'.join(u'<p>%s</p>' % p.replace('\n', '<br>\n') for p in _paragraph_re.split(escape(value)))
+    result = u'\n\n'.join(u'<p>%s</p>' % p.replace('\n', '<br>') for p in _paragraph_re.split(escape(value)))
```

Comment:
It might be correct, but why are we joining with `u'\n\n'`? Isn't this method replacing \n with `<br>`?